Mathlib/NumberTheory/DiophantineApproximation.lean

```diff
@@ -50,7 +50,7 @@ Both statements are combined to give an equivalence,
 There are two versions of Legendre's Theorem. One, `Real.exists_rat_eq_convergent`, uses
 `Real.convergent`, a simple recursive definition of the convergents that is also defined
 in this file, whereas the other, `Real.exists_continued_fraction_convergent_eq_rat`, uses
```

Comment:
For consistency with the `convergents` -> `convs` renaming in this PR, I'm tempted to suggest renaming `convergent` to `conv`. Thinking about it a bit more though, `conv` might be an overloaded abbreviation (e.g. we have `conv` mode). I guess for now we can leave these alone, but cc: @MichaelStollBayreuth in case he has any opinions.

Ah, I just noticed that `continued_fraction_conv_eq_convergent` and `exists_continued_fraction_convergent_eq_rat` don't seem to follow our [naming conventions](https://leanprover-community.github.io/contribute/naming.html) (if I'm not confused they should have contained `generalizedContinuedFraction` somewhere in the name). Maybe this should just be mentioned in a TODO and left for a follow-up PR?

Reply:
We shouldn't remain tech debts as possible.
I renamed `Real.convergent` to `Real.convs` & related theorems.

Reply:
I'd actually prefer to keep `Real.convergent` as the name, as its meaning should be clear, whereas `Real.convs` is less so. (In the general continued fractions context, the (fully qualified) name includes a reference to continued fractions, so this is less of a concern, but here it does not.) Also, length of names is not really an issue in `Mathlib.NumberTheory.DiophantineApproximation` except for the few names referring to general continued fractions stuff (which obviously need to be consistent with the changes there). (The violation of the naming convention must be due to the port, I think.)

Reply:
@MichaelStollBayreuth I have reverted the change about `Real.convs`, sorry.